R/irods-demo.R

```diff
@@ -264,8 +283,10 @@ local_create_irods <- function(
   withr::defer(setwd(old_dir), envir = env)
 
   # switch to new iRODS project
-  create_irods(host, zone_path, overwrite = TRUE)
+  create_irods(host, overwrite = TRUE)
   withr::defer(unlink(path_to_irods_conf()), envir = env)
 
   invisible(dir)
 }
+
+rirods_host <- function() "http://localhost:9001/irods-http-api/0.2.0"
```

Comment:
Why is this a function instead of just a variable?

Reply:
this variable is now loaded on attach